csm/plugins/cortx/health.py

```diff
@@ -178,17 +178,11 @@ def _parse_alert(self, message):
                     self._prepare_resource_key(resource_schema)
             health_value = self._derive_health(message, resource_schema\
                 [const.HEALTH_ALERT_TYPE])
-            """
-            This a special case so handling seperatly.
-            """
-            if resource_type == const.SAS_RESOURCE_TYPE:
-                self._handle_sas_alert(resource_schema, health_value, extended_info)
-            else:
-                resources[const.DURABLE_ID] = info.get(const.ALERT_RESOURCE_ID, "")
-                resources[const.KEY] = resource_type + "-" + \
-                    info.get(const.ALERT_RESOURCE_ID, "")
-                resources[const.ALERT_HEALTH] = health_value
-                resource_schema[const.RESOURCE_LIST].append(resources)
+            resources[const.DURABLE_ID] = info.get(const.ALERT_RESOURCE_ID, "")
+            resources[const.KEY] = resource_type + "-" + \
```

Comment:
Make use of f strings or Format instead of +

Reply:
Done